tests/test_traits.py

```diff
@@ -0,0 +1,166 @@
+from __future__ import annotations
+from abc import ABC
+from dataclasses import dataclass
+from typing import Annotated
+
+import pytest
+
+from xdsl.ir import OpResult, OpTrait, Operation
+from xdsl.irdl import Operand, irdl_op_definition
+from xdsl.utils.exceptions import VerifyException
+from xdsl.dialects.builtin import IntegerType, i1, i32, i64
+
+
+@dataclass(frozen=True)
+class LargerResultTrait(OpTrait):
+    """Check that the only result has a larger bitwidth than the operand."""
+
+    def verify(self, op: Operation) -> None:
+        # This function is never called in this test
```

Comment:
Why not implement it and call it? xD
I see that it's somewhat useless, but having it here like this looks weird.
If only we had a testing infrastructure that gave easy and simple ways to define test stubs....

Reply:
Well, it just add more code and tests that do not bring anything new :/
I prefer keeping minimal tests, what do you think?